bin/gl-style-validate.ts

```diff
@@ -2,7 +2,7 @@
 
 import minimist from 'minimist';
 import rw from 'rw';
-import validate from '../src/validate_style';
+import {validateStyle as validate} from '../src/validate_style';
```

Comment:
Can these renames be removed as well?

Reply:
Yes, the src/index.ts I imagine will be breaking if I remove, but I can look into the rest